### ISSUE TITLE
Fix backporting issues

### ISSUE THREAD

**Issue by danyx23** [owid/owid-grapher#1870]

**Comment by larsyencken** [comment]
@danyx23 Can you have a go at splitting off the parts of this you reasonably expect into this cycle, given the Data Pages discovery work you're also doing?

My sense is that this is still two cycles worth of work. I wonder if there's a good intermediate goal to aim for for thus cycle, somewhere where it was already useful for authors and you were happy to leave it there for a while.

I say that because next cycle we'll probably need time and energy from you and Marcel both on onboarding Sophia and Christian, so we'll probably need to choose whether to do a small amount more here or to focus more on data pages. After that you'll be soon on leave, so worst-case it could be mid-year before you get to the final round of work here.

**Comment by larsyencken** [comment]
I take it back, you've already scope this to this cycle. Have you estimated the items on your to-do list to see if it matches your overall 3-week estimate 😅

**Comment by danyx23** [comment]
@larsyencken yeah I think the bulk of the work will be the stuff in the Metadata/Workflow section. The missing translation counts on Ike developing all of these components and then only the data migration being left which should be straightforward. The GDoc mapping issues might be a list that increases drastically as we see more examples. This part is the one that will probably spill over but not make a full cycle? It might also be something that Matthieu or Ike could do.

The timing might work out quite fine - as long as the main infrastructure is in place (the metadata/workflow items), we could accept a less than excellent backporting for a while because authors would manually convert them anyhow. Only when we do the bulk backporting of all content to turn off wordpress do we need a reasonably high fidelity backporting to be in place.

**Comment by larsyencken** [comment]
No sweat, could be useful to keep track of the things that come up elsewhere, e.g. in the tracking issue: https://github.com/owid/owid-grapher/issues/1735

**Comment by larsyencken** [comment]
Closing this now, we already use this in production.
